data/json/terrain.json

```diff
@@ -2736,6 +2741,7 @@
         "type" : "terrain",
         "id" : "t_window_frame",
         "name": "window frame",
+        "description": "A wooden window frame that has shattered glass around it.  Some of those shards look jagged and harmful if you tried to climb through.  You could smash the rest out loudly, or you could clean the window of the broken pieces quietly but it'd take longer.",
```

Comment:
> Some of those shards look jagged and harmful if you tried to climb through

Nit: consider rewording. They'd be jagged and harmful even if you didn't try to climb through.

> You could smash the rest out loudly, or you could clean the window of the broken pieces quietly but it'd take longer

Second phrase is quite wordy, consider rewording. Maybe something like:

> You could smash out the remaining pieces, or take your time and quietly clean them up.